File: 23-9-26/1.py
import numpy as np
def rms(a): #define a function
    return np.sqrt(np.mean(a ** 2)) #first it takes mean of the entered values then it takes sq root
n = int(input())
a = np.array(list(map(float, input().split())))  #input.split seperates the elements entered when there is spce between them# map converts the strings into actual decimal points
#list holds the values and np.arry converts it into a array

print(f"{rms(a):.2f}")

File: 23-9-26/1.c
#include <stdio.h>
#include <math.h>
double norm(double a[], int n)
{
    double sum = 0;

    for (int i = 0; i < n; i++)
    {
        sum += a[i] * a[i];
    }

    return sqrt(sum);
}

double rms(double a[], int n)
{
    return norm(a, n) / sqrt(n);
}

int main()
{
    double a[] = {3, 4, 0, 5};
    int n = 4;

    printf("For the array {3,4,0,5}: %.2f\n", rms(a, n));
    double  b[] = {1,-1,1,-1};
    n = 4;
     printf("For the array {-1,1,-1,1}: %.2f\n", rms(b, n));

    double c[] = {7.5};
    n=1;
    printf("For the array {7.5}: %.2f\n", rms(c, n));    
    return 0;
}
